src/modules/Web3Module.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  Module,
+  VuexModule,
+  Mutation } from 'vuex-module-decorators'
+import Web3 from 'web3'
+import { HttpProvider } from 'web3/providers'
+
+@Module({ namespaced: true, name: 'web3Module' })
+export default class Web3Module extends VuexModule {
+
+  public web3: Web3 = Object()
```

Comment:
is the throw-away val needed?

Reply:
The Typescript vuex modules need the property initialized with a non-null/non-undefined value to work correctly.  Stuff like `public web3!: Web3` doesn't work - run into state management problems later.  Still looking into it...